doc/contributing/tests.md

```diff
@@ -134,6 +134,7 @@ Recommended way to run acc tests is to use two distinct credentials:
 one for terraform related actions, and one for driftctl scan.
 
 You can override environment variables passed to terraform operations by adding `ACC_` prefix on env variables.
+Environment used in `PreCheck()` is using using overriden env too as it is common to perform mutations in that place.
```

Comment:
I don't understand your sentence here, can you help me get it in order to rephrase it well.

Reply:
`In our acceptance tests, we may need read/write permissions during specific contexts (e.g. terraform init, apply, destroy) or lifecycle (PreExec and PostExec). If needed, you can override environment variables in those contexts by adding `ACC_` prefix on env variables.`

Reply:
This is a way better, fixed